### DISCUSSION TITLE
vai let's try for answer badge

### DISCUSSION THREAD

**Discussion by hafijul233** [hafijul233/HackerRank#2] [Q&A] [answered]
vai ami ek ta question liklam apni answer den. ami anwser hisabe mark korbo .dheki badge pai ki na?

**Comment by mah-shamim** [comment, marked as answer]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

    Ask questions you’re wondering about.
    Share ideas.
    Engage with other community members.
    Welcome others and are open-minded. Remember that this is a community we
    build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.
